packages/backend-core/src/sql/sql.ts

```diff
@@ -813,30 +799,66 @@ class InternalBuilder {
     return query
   }
 
+  isSqs(t?: Table): boolean {
```

Comment:
NIT: I am not too fan of these optional parameters in here, as it might be easy to miss in some refactors or just while using it. This allows the caller to use `isSqs()`, skipping the parameter, so it's not obvious if it's meant to be passed or not.
Also, 1-letter parameter in functions makes the code harder to follow

Reply:
This did start life having no parameter at all but there's an annoying, singular case where the table that's being checked is _not_ `this.table`, which made me make this take a param and then because most of the uses would pass `this.table` it felt a bit nicer to make this optional.

The 1 letter param name is just because if I call it `table` I then have to call the const a little below it something else and that felt tedious.

Happy to go with another way of doing it if you have any suggestions.

Reply:
I see the reasoning. Maybe this might be cleaned with a "private method" that requires the table, and a public one that will use the `privateIsSqs(this.table)`?